.jenkins/pipelines/Azure/Windows/Jenkinsfile

```diff
@@ -122,21 +123,30 @@ try{
                 "Win2019 Debug Cross Compile with DCAP libs":             { windowsCrossCompile('acc-win2019', 'Debug', 'ON') },
                 "Win2019 Release Cross Compile with DCAP libs":           { windowsCrossCompile('acc-win2019', 'Release', 'ON') },
                 "Win2019 Debug Cross Compile DCAP LVI":                   { windowsCrossCompile('acc-win2019', 'Debug', 'ON', 'ControlFlow') },
-                "Win2019 Release Cross Compile DCAP LVI":                 { windowsCrossCompile('acc-win2019', 'Release', 'ON', 'ControlFlow') }
+                "Win2019 Release Cross Compile DCAP LVI":                 { windowsCrossCompile('acc-win2019', 'Release', 'ON', 'ControlFlow') },
+
```

Comment:
I don't have a good sense of whether that's an interesting set of builds for Windows, if I should pick some others instead/too. Some feedback from someone who understands what's most often used would be very welcome here.

Reply:
We should probably add a DEBUG config as well
"Win2019 Debug Cross Compile DCAP LVI":                 { windowsCrossCompile('acc-win2019', 'Debug', 'ON', 'ControlFlow') },

Reply:
I think the ones you have added are good. Can we add "Win2019 Sim Debug Cross Compile LVI snmalloc":         { windowsCrossCompile('acc-win2019', 'Debug', 'OFF', 'ControlFlow', '1', 'OFF', ['-DUSE_SNMALLOC=ON']) }, instead of the 2019 simulation mode release config below?

Reply:
Of course, done.